internal/erofsutils/mount_linux.go

```diff
@@ -43,6 +44,98 @@ func ConvertTarErofs(ctx context.Context, r io.Reader, layerPath string, mkfsExt
 	return nil
 }
 
+// prepareInputFile takes a reader and returns a seekable file along with a flag
+// indicating if the file is temporary and should be cleaned up afterward.
+func prepareInputFile(ctx context.Context, r io.Reader) (file *os.File, isTemp bool, err error) {
```

Comment:
prepareSeekableTarFile?
`isTemp` -> `needClose`?

Reply:
.. the name `prepareInputFile` is too generic, I think `prepareSeekableTarFile` is better, but I have no strong opinion on this.